### DISCUSSION TITLE
Warning: imap_open(): Couldn't open stream outlook.office365.com:993/imap/ssl}INBOX

### DISCUSSION THREAD

**Discussion by Kamekshakumar** [uvdesk/community-skeleton#663] [Q&A] [answered]
Hi Guys,

I am facing below issue while configuring office 365 support email in uvdesk.

Can anyone help?

![uvdesk error](https://github.com/uvdesk/community-skeleton/assets/15952829/8d1fac79-fa8c-43ff-ae63-925dd2e4e0d0)

**Comment by komal-sh-27** [comment, marked as answer]
> I am facing below issue while configuring office 365 support email in uvdesk.

If you are using **microsoft office 365 modern auth** email, then **stable release V-1.1** is not working **with microsoft auth**. We have changes for the modern auth email, so this feature will be added with our next release in V-1.2.0 or most renamed V-2.0 till this version is in development mode. 

So, you will be install new project with the dev master command and setup microsoft apps with modern auth outook.

Please, You can try out the newest available version using the following command:

**_$ composer create-project uvdesk/community-skeleton:dev-master;_**

We would request you try the new updates and let us know if you come across any notable issues which we can address right away during our testing phase.
